### PR TITLE
feat : Implement quick create descussion chat drawer - EXO-60885

### PR DESCRIPTION
**Pull request by sofyenne**
We will implement a drawer which will create a quick chat discussion.

### CODE REVIEW

File: application/src/main/webapp/css/components/chatDrawer.less [hunk 1]
```diff
@@ -442,3 +442,22 @@ input#taskDueDate:focus {
 input.add-room-name:focus {
   border-color: @focusInputColor;
 }
+#Quicksuggester {
+  .v-autocomplete.identitySuggester {
+    .v-input__slot {
+      margin: auto !important;
+    }
+
```
Comment (azayati) [inline application/src/main/webapp/css/components/chatDrawer.less]: remove empty line

File: application/src/main/webapp/css/components/chatDrawer.less [hunk 2]
```diff
@@ -442,3 +442,22 @@ input#taskDueDate:focus {
...
+    .v-select__selections {
+      flex-wrap: nowrap;
+      white-space: nowrap;
+      overflow: hidden;
+      margin: auto;
+      display: flex !important;
+    }
+
```
Comment (azayati) [inline application/src/main/webapp/css/components/chatDrawer.less]: remove empty line

File: application/src/main/webapp/vue-app/components/initComponents.js [hunk 3]
```diff
@@ -38,7 +40,10 @@ const components = {
   'exo-content-editable': ExoContentEditable,
   'exo-chat-drawer': ExoChatDrawer,
   'popover-chat-button': PopoverChatButton,
-  'exo-chat-space-settings': spaceChatSetting
+  'exo-chat-space-settings': spaceChatSetting,
+  'exo-chat-quick-create-discussion-drawer': ExoChatQuickCreateDiscussionDrawer,
+  'exo-chat-quick-discussion-participant-item': ExoChatQuickDiscussionParticipantItem,
+
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/initComponents.js]: remove empty line

File: application/src/main/webapp/vue-app/components/modal/ExoChatQuickCreateDiscussionDrawer.vue [hunk 4]
```diff
@@ -0,0 +1,130 @@
+<template>
+  <exo-drawer
+    ref="QuickCreateDiscussionDrawer"
+    id="QuickCreateDiscussionDrawer"
+    right
+    @closed="close">
+    <template slot="title">
+      <span class="PopupTitle popupTitle"> <v-icon left @click="close">mdi-arrow-left</v-icon>{{ $t('exoplatform.chat.quick.create.discussion') }}</span>
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/modal/ExoChatQuickCreateDiscussionDrawer.vue]: redondant PopupTitle css class

File: services/src/main/resources/locale/portlet/chat/Resource_en.properties [hunk 5]
```diff
@@ -63,6 +63,9 @@ exoplatform.chat.close=Close
 exoplatform.chat.compareddate.invalid.message=Start date cannot be greater than end date!
 exoplatform.chat.create.team=Create a new room
 exoplatform.chat.quick.create.discussion=Add a new discussion
+exoplatform.chat.quick.create.discussion.info=If you choose more than a user,you'll have to name a room
```
Comment (azayati) [inline services/src/main/resources/locale/portlet/chat/Resource_en.properties]:
```suggestion
exoplatform.chat.quick.create.discussion.info=If you choose more than a user, you'll have to name a room
```

File: services/src/main/resources/locale/portlet/chat/Resource_en.properties [hunk 6]
```diff
@@ -63,6 +63,9 @@ exoplatform.chat.close=Close
...
+exoplatform.chat.quick.create.discussion.team.people=Add people
```
Comment (azayati) [inline services/src/main/resources/locale/portlet/chat/Resource_en.properties]:
```suggestion
exoplatform.chat.quick.create.discussion.add.people=Add people
```

File: services/src/main/resources/locale/portlet/chat/Resource_en.properties [hunk 7]
```diff
@@ -63,6 +63,9 @@ exoplatform.chat.close=Close
...
+exoplatform.chat.quick.create.discussion.placeholder=Tap your room name
```
Comment (azayati) [inline services/src/main/resources/locale/portlet/chat/Resource_en.properties]:
```suggestion
exoplatform.chat.quick.create.discussion.room.placeholder=Tape your room name
```

File: application/src/main/webapp/vue-app/components/modal/ExoChatQuickCreateDiscussionDrawer.vue [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+        this.participantItem.splice(index, 1);
+      }
+    },
+  }
+};
+</script>
+
+
+
+
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/modal/ExoChatQuickCreateDiscussionDrawer.vue]: remove empty lines

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: application/src/main/webapp/vue-app/components/modal/ExoChatQuickCreateDiscussionDrawer.vue (hunk 8 above)
Comment (sofyenne) [reply application/src/main/webapp/vue-app/components/modal/ExoChatQuickCreateDiscussionDrawer.vue]: removed !

File: services/src/main/resources/locale/portlet/chat/Resource_en.properties (hunk 7 above)
Comment (sofyenne) [reply services/src/main/resources/locale/portlet/chat/Resource_en.properties]: done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**